macros/edr/tests/test_utils/get_model_graph_node.sql

```diff
@@ -4,11 +4,11 @@
         {% set test_graph_node = model %}
         {% set test_depends_on_unique_ids = test_graph_node.depends_on.nodes %}
         {# model relation is the relation object of the model where the test is defined #}
-        {% set model_name = model_relation.name | lower %}
+        {% set model_node = elementary.get_node_by_relation(model_relation) %}
         {% set depends_on_nodes = elementary.get_nodes_by_unique_ids(test_depends_on_unique_ids) %}
         {% if depends_on_nodes %}
             {% for node in depends_on_nodes %}
-                {% if node.name | lower == model_name %}
+                {% if node.unique_id == model_node.unique_id %}
```

Comment:
Here we are already iterating over the depends_on_nodes list and compare each depends on node name to the model relation name. I would suggest adding here additional check that will validate if either the name is matched or the alias of the node is matched. For example, it could be something like - 
<img width="551" alt="image" src="https://user-images.githubusercontent.com/4517234/180489184-5e016fa5-f8e1-4956-be25-a57fdd2a70f2.png">

(Please note that alias does not always exist in a dbt graph node so this is why I used 'get' when accessing it)